LayoutTests/fast/reflections/reflection-removed-crash-2.html

```
@@ -0,0 +1,20 @@
+<style>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)